src/gepetto_viewer_rerun/client.py

```diff
@@ -0,0 +1,350 @@
+from dataclasses import dataclass
+import logging
+import numpy as np
+from math import tau
+from enum import Enum
+from typing import List, Union
+import rerun as rr
+
+from .entity  import (
+    Entity,
+    MeshFromPath
+)
+from .scene import Scene
+
+logger = logging.getLogger(__name__)
+
+
+class Archetype(Enum):
+    ARROWS3D = 0
+    BoxeS3D = 1
+    CAPSULES3D = 2
+    LINESTRIPS3D = 3
+    MESH3D = 4
+    MESH_FROM_PATH = 5
+    POINTS3D = 6
+    
+
+@dataclass
+class Client:
+    def __init__(self):
+        self.gui = Gui()
+
+@dataclass
```

Comment:
```suggestion
class Client:
    def __init__(self):
        self.gui = Gui()

```

dataclass only make sense for classes that only (or almost only) define some attributes